uniparc-rest/src/main/java/org/uniprot/api/uniparc/controller/UniParcController.java

```diff
@@ -377,6 +372,71 @@ public ResponseEntity<MessageConverterContext<UniParcEntry>> searchByProteomeId(
         return super.getSearchResponse(results, getByUpIdRequest.getFields(), request, response);
     }
 
+    @SuppressWarnings("squid:S6856")
+    @GetMapping(
+            value = "/proteome/{upId}/stream",
+            produces = {
+                TSV_MEDIA_TYPE_VALUE,
+                FASTA_MEDIA_TYPE_VALUE,
+                LIST_MEDIA_TYPE_VALUE,
+                APPLICATION_XML_VALUE,
+                APPLICATION_JSON_VALUE,
+                XLS_MEDIA_TYPE_VALUE,
+                RDF_MEDIA_TYPE_VALUE,
+                TURTLE_MEDIA_TYPE_VALUE,
+                N_TRIPLES_MEDIA_TYPE_VALUE
+            })
+    @Operation(
+            hidden = true,
+            summary = PROTEOME_UPID_STREAM_UNIPARC_OPERATION,
+            description = PROTEOME_UPID_STREAM_UNIPARC_OPERATION_DESC,
+            responses = {
+                @ApiResponse(
+                        content = {
+                            @Content(
+                                    mediaType = APPLICATION_JSON_VALUE,
+                                    array =
```

Comment:
Can you please merge with master to also reflect the recent changes about OpenApi?
                            @Content(
                                    mediaType = APPLICATION_JSON_VALUE,
                                    schema = @Schema(implementation = StreamResult.class)),